controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -1767,13 +1767,27 @@ int cli_bml::set_dcs_continuous_scan_enable(const std::string &radio_mac, int8_t
 {
     std::cout << "set_dcs_continuous_scan_enable" << std::endl;
 
+    int ret = bml_set_dcs_continuous_scan_enable(ctx, radio_mac.c_str(), enable);
+
+    printBmlReturnVals("bml_set_dcs_continuous_scan_enable", ret);
+
     return 0;
 }
 
 int cli_bml::get_dcs_continuous_scan_enable(const std::string &radio_mac)
 {
     std::cout << "get_dcs_continuous_scan_enable" << std::endl;
 
+    int enable = -1;
+    int ret    = bml_get_dcs_continuous_scan_enable(ctx, radio_mac.c_str(), &enable);
+
+    if (ret == BML_RET_OK) {
+        std::cout << "dcs-continuous-scan-enable=" << ((enable == 1) ? "True" : "False")
```

Comment:
string_utils::bool_str(enable);

Reply:
wasn't familiar with it - very useful - will fix, thanks